webapp/src/test/resources/application-test.yml

```diff
@@ -1,14 +1,10 @@
 spring:
   datasource:
-    url: jdbc:mysql://localhost:3306/warehouse_test?useSSL=false&useLegacyDatetimeCode=false&characterEncoding=utf8
+    url-schema: warehouse_test
```

Comment:
... instead of copying & pasting and then maintaining the full connection string, this lets just the one variable be set.